### ISSUE TITLE
Detect when Chromecast turns off

### ISSUE THREAD

**Issue by mhalano** [masmu/pulseaudio-dlna#234]
When I turn off Chromecast and keep pulseaudio-dlna command running, the Chormecast keeps appearing on sound properties on Ubuntu. May be should see if Chromecast is alive and if is not remove the sink ins instead of just remove the sink when pulseaudio-dlna closes?

**Comment by masmu** [comment]
Yeah this should be done absolutely. What is your method of shutting the device down? In the Android _Chromecast_ application I just find the option to restart the device.

**Comment by mhalano** [comment]
@masmu I just change the TV source (to cable instea HDMI for instance) or turn off the TV. Since Chromecast dosen't respond it should be removed from audio menu. What do you think?
